openshift_images/configuring-samples-operator.adoc

```diff
@@ -8,6 +8,21 @@ toc::[]
 
 The Cluster Samples Operator, which operates in the `openshift` namespace, installs and updates the {op-system-base-full}-based {product-title} image streams and {product-title} templates.
 
+[IMPORTANT]
+.Cluster Samples Operator is being downsized  
+====
+. No RFEs will be accepted for whole of samples operator
+
+. For Non-s2i(Non-Builder Imagestreams/Templates)  
+- No new Imagestreams /Templates 
+- No updates to the existing Imagestreams/templates
```

Comment:
I would say not updates to imagestream and templates unless is due to a CVE

Reply:
got it changed